core/templatetags/discover_uni_tags.py

```diff
@@ -52,3 +54,18 @@ def map_distance_learning_values(key, language):
     if key in DICT.get('distance_learning_values'):
         return DICT.get('distance_learning_values').get(key).get(language)
     return key
+
+
+@register.simple_tag
+def should_show_accordion(course, accordion_type):
+    if accordion_type == STUDENT_SATISFACTION_KEY:
+        return course.show_satisfaction_stats
+    elif accordion_type == ACCREDITATION_KEY:
```

Comment:
Just a minor point - the elif is not needed here, but fine with if you think is more readable.